pkg/sql/pgwire/command_result.go

```diff
@@ -124,9 +125,15 @@ func (r *commandResult) Close(t sql.TransactionStatusIndicator) {
 		r.typ == commandComplete &&
 		r.stmtType == tree.Rows {
 
-		r.err = errors.Errorf("execute row count limits not supported: %d of %d",
+		r.err = pgerror.UnimplementedWithIssueErrorf(4035,
+			"execute row count limits not supported: %d of %d",
 			r.limit, r.rowsAffected)
-		r.conn.bufferErr(convertToErrWithPGCode(r.err))
+		// We have to do the telemetry explicitly here because automatic
+		// telemetry for unimplemented features is only done in the
+		// connExecutor.
+		telemetry.Count("errorcodes." + pgerror.CodeFeatureNotSupportedError)
```

Comment:
I'm missing something -- why can't we let the regular unimplemented counting get this?

Reply:
Like I said, I'm probably missing something. Usually we let the error bubble up to the conn executor and it inspects it as a goes back to the client, so I was wondering why that didn't work for this case?

Reply:
This was actually pretty painless. I had forgotten that I had dropped the dependency on the executor in a previous PR. Done.